Content.Server/AI/Operators/Inventory/DropHandItemsOperator.cs

```diff
@@ -0,0 +1,26 @@
+using Content.Server.GameObjects;
+using Robust.Shared.Interfaces.GameObjects;
+
+namespace Content.Server.AI.Operators.Inventory
+{
+    public class DropHandItemsOperator : AiOperator
+    {
+        private readonly IEntity _owner;
+
+        public DropHandItemsOperator(IEntity owner)
+        {
+            _owner = owner;
+        }
+
+        public override Outcome Execute(float frameTime)
+        {
+            _owner.TryGetComponent(out HandsComponent handsComponent);
```

Comment:
Weird use of `TryGet`, since `handsComponent` is used non-null.

Reply:
Uhhh NFI why I did that, fixed.